src/magic_folder/snapshot.py

```diff
@@ -628,60 +600,51 @@ def write_snapshot_to_tahoe(snapshot, author_key, tahoe_client):
     # upload the content itself
     content_cap = yield tahoe_client.create_immutable(snapshot.get_content_producer())
 
-    # sign the snapshot (which can only happen after we have the content-capability)
-    author_signature = sign_snapshot(author_key, snapshot, content_cap)
-    author_signature_base64 = base64.b64encode(author_signature.signature)
-    author_data = snapshot.author.to_remote_author().to_json()
-
-    author_cap = yield tahoe_client.create_immutable(
-        json.dumps(author_data)
+    # create our metadata
+    snapshot_metadata = {
+        "snapshot_version": SNAPSHOT_VERSION,
+        "name": snapshot.name,
+        "author": snapshot.author.to_remote_author().to_json(),
+        "parents": [
+            parent_cap.encode("utf8")
+            for parent_cap in parents_raw
+        ]
+    }
+    metadata_cap = yield tahoe_client.create_immutable(
+        json.dumps(snapshot_metadata)
```

Comment:
Not necessarily for this PR but the metadata and content could be uploaded in parallel instead of serially.

Reply:
True. Not sure if it'll make much difference because Tahoe does uploads in serial (but downloads in parallel). However, maybe the WebUI returns before all the data has been uploaded to all the storage-servers? (I don't recall).